smartapps/tslagle13/thermostat-mode-director.src/i18n/bg-BG.properties

```diff
@@ -77,3 +77,4 @@
 '''Next page'''=Следваща страница
 '''Text'''=Текст
 '''Number'''=Номер
+'''Here you can setup the ability to 'boost' your thermostat. In the event that your thermostat is 'off' and you need to heat or cool your home for a little bit you can 'touch' the app in the 'My Apps' section to boost your thermostat'''=Тук може да настроите възможността за подобряване на термостата. В случай че термостатът е изключен и се налага да затоплите или охладите дома си за малко, може да докоснете приложението в раздела „Моите приложения“, за да подобрите термостата.
```

Comment:
`thermostat. In the event`. In the code, there is two spaces between the period and `In`

Reply:
also, the code has ` and you need to heat or cool your your home`. your is repeated twice

Reply:
also, this line ends with a period in the code